docs/workflows/build-pages-with-storybook.md

```diff
@@ -52,7 +53,8 @@ In such cases it is natural to use [args composition](../writing-stories/args.md
   paths={[
     'react/page-story-with-args-composition.js.mdx',
     'react/page-story-with-args-composition.ts.mdx',
-    'vue/page-story-with-args-composition.js.mdx'
+    'vue/page-story-with-args-composition.js.mdx',
+    'angular/page-story-with-args-composition.ts.mdx'
```

Comment:
```suggestion
    'angular/page-story-with-args-composition.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)